skimage/future/graph/rag.py

```diff
@@ -263,6 +266,125 @@ def rag_mean_color(image, labels, connectivity=2, mode='distance',
            http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.11.5274
 
     """
+
+
+    def define_graph(graph,labels,image, extra_arguments=[],
```

Comment:
Why does this method have a generic name ? It is doing something very specific, ie, defining a color similarity based RAG.